front/src/Stores/BiggestAvailableAreaStore.ts

```diff
@@ -1,112 +1,136 @@
-import { get, writable } from "svelte/store";
+import { writable } from "svelte/store";
 import type { Box } from "../WebRtc/LayoutManager";
 import { HtmlUtils } from "../WebRtc/HtmlUtils";
-import { LayoutMode } from "../WebRtc/LayoutManager";
-import { embedScreenLayout } from "./EmbedScreensStore";
 
 /**
  * Tries to find the biggest available box of remaining space (this is a space where we can center the character)
  */
 function findBiggestAvailableArea(): Box {
     const game = HtmlUtils.querySelectorOrFail<HTMLCanvasElement>("#game canvas");
-    if (get(embedScreenLayout) === LayoutMode.VideoChat) {
-        const children = document.querySelectorAll<HTMLDivElement>("div.chat-mode > div");
-        const htmlChildren = Array.from(children.values());
-
-        // No chat? Let's go full center
-        if (htmlChildren.length === 0) {
-            return {
-                xStart: 0,
-                yStart: 0,
-                xEnd: game.offsetWidth,
-                yEnd: game.offsetHeight,
-            };
-        }
+    // map iframes to Box'es and remove ones outside of the game canvas space
+    const iframeBoxes: Box[] = Array.from(document.getElementsByTagName("iframe"))
```

Comment:
Warning, some iframes should not be taken into account. We don't want to take into account the iframes that are directly embedded into maps.